test/ship_test.rb

```diff
@@ -0,0 +1,45 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/ship.rb'
+
+class PlayerTest < Minitest::Test
+
+  def test_ship_exists
+    ship = Ship.new("Cruiser", 3)
+
+    assert_instance_of Ship, ship
+  end
+
+  def test_ship_instantiation
+    ship = Ship.new("Cruiser", 3)
+
+    assert_equal "Cruiser", ship.name
+    assert_equal 3, ship.length
+    assert_equal 3, ship.health
+  end
+
+  def test_ship_full_health_has_sunk
+    ship = Ship.new("Cruiser", 3)
+
+    assert_equal false, ship.sunk?
+  end
+
+  def test_ship_can_hit
```

Comment:
prolly semantics, but it's more "ship_can_be_hit", rather than whether the ship itself can hit anything